fast64_internal/f3d/f3d_gbi.py

```diff
@@ -4291,8 +4291,14 @@ def geoFlagListToWord(flagList, f3d):
 
 @dataclass(unsafe_hash=True)
 class SPGeometryMode(GbiMacro):
-    clearFlagList: list
-    setFlagList: list
+    clearFlagList: set[str] = field(default_factory=set)
+    setFlagList: set[str] = field(default_factory=set)
+
+    def extend(self, clear_list: set, set_list: set):
+        clear_list = self.clearFlagList | clear_list
+        set_list = self.setFlagList | set_list
+        self.setFlagList = set_list - clear_list
+        self.clearFlagList = clear_list - set_list
```

Comment:
per gbi.h,

> \*	One gSPGeometryMode(pkt,c,s) GBI is equal to these two GBIs.
> \*
> \*		gSPClearGeometryMode(pkt,c)
> \*		gSPSetGeometryMode(pkt,s)

that means the "set" flags take precedence, so shouldn't this be
```suggestion
    def extend(self, clear_list: set, set_list: set):
        clear_list = self.clearFlagList | clear_list
        set_list = self.setFlagList | set_list
        self.setFlagList = set_list
        self.clearFlagList = clear_list - set_list
```
(idk)